blst/Cargo.toml

```diff
@@ -40,6 +41,18 @@ parallel = [
 ]
 minimal-spec = ["kzg/minimal-spec", "kzg-bench/minimal-spec"]
 
+# By default, compile with ADX extension if the host supports it.
+# Binary can be executed on systems similar to the host.
+blst-default = ["blst/default"]
```

Comment:
This is no-op because `default-features = false` is not specified for `blst` dependency.

`blst-default` should simply be removed and most likely other features as well. `blst` maintainer is stubborn and refuses to follow idiomatic Rust conventions, so essentially none of the below should be necessary in practice, it'll use features that target CPU supports automatically, so just use corresponding `RUSTFLAGS="-C target-cpu=X"` where `X` is target CPU to engage necessary features automatically.